lib/lexisnexis/verification_error_parser.rb

```diff
@@ -41,15 +45,25 @@ def base_error_message
       "#{tracking_ids} Verification failed with code: '#{error_code}'"
     end
 
-    def product_error_messages
+    def parse_product_error_messages
       products = body['Products']
       return { products: 'Products missing from response' } if products.nil?
 
       products.each_with_object({}) do |product, error_messages|
-        next if product['ProductStatus'] == 'pass'
-
-        if dob_year_only? && product['ProductType'] == 'InstantVerify'
-          next if instant_verify_dob_year_only_pass?(product['Items'])
+        if product['ProductType'] == 'InstantVerify'
+          original_passed = (product['ProductStatus'] == 'pass')
+          passed_partial_dob = instant_verify_dob_year_only_pass?(product['Items'])
+
+          Rails.logger.info({
```

Comment:
this will go to `production.log` instead of `events.log` but I think that's fine?

Reply:
if it's temporary for evaluation, yeah I think so. Ideally it gets incorporated into the step event results, but I'm not sure if that's possible right now.